api/migrations/versions/ef52e75a2aec_.py

```diff
@@ -0,0 +1,38 @@
+"""empty message
```

Comment:
migration file should have a name after the underscore and that name should be in the message

Reply:
resolved